PWG/CaloTrackCorrBase/AliCaloTrackReader.cxx

```diff
@@ -2391,6 +2392,25 @@ Bool_t AliCaloTrackReader::FillInputEvent(Int_t iEntry, const char * /*curFileNa
     else                        fhNEventsAfterCutCen->Fill(15.5,cen);
   }
 
+  //----------------------------------------------------------------
+  //Check if there is an event plane angle value, PbPb analysis,
+  // and if an event plane angle bin selection is requested
+  //----------------------------------------------------------------
+  
+  Float_t ep =  GetEventPlaneAngle();
+  if ( fEventPlaneBin[0] >= 0 && fEventPlaneBin[1] >= 0 )
+  {
+    AliDebug(1,Form("Angle %f in [%f,%f]?", ep, fEventPlaneBin[0], fEventPlaneBin[0]));
+
+    if ( ep >= fEventPlaneBin[0] || ep <  fEventPlaneBin[0]   )  return kFALSE; //reject events out of bin.
```

Comment:
I guess you mean
if ( ep >= fEventPlaneBin[0] || ep < fEventPlaneBin[1] )
The maximum is not considered